ui/index.d.ts

```diff
@@ -553,27 +565,63 @@ export interface IAzureQuickPickOptions extends QuickPickOptions {
     suppressPersistence?: boolean;
 }
 
+export interface ISubWizardOptions<T> {
+    /**
+     * The steps to prompt for user input, in order
+     */
+    promptSteps: AzureWizardPromptStep<T>[];
+
+    /**
+     * The steps to execute, in order
+     */
+    executeSteps?: AzureWizardExecuteStep<T>[];
+}
+
+export interface IWizardOptions<T> extends ISubWizardOptions<T> {
```

Comment:
So weird to me that the main class extends the sub class lol

Reply:
I see what you mean - it's weird because I'm talking about the sub-wizard, not the sub-class. Although these are both interfaces, not classes lol. Anyways let me know if you think of a better name